HPC/README.md

```diff
@@ -485,3 +485,15 @@ GCC	| 11	| -O3 -mcpu=neoverse-v1	| -fopenmp	| -ffast-math
 CLang/LLVM |	14 | -O3 -mcpu=neoverse-512tvb	| -fopenmp	| -ffast-math
 Arm Compiler for Linux |	23.04 |	-O3 -mcpu=neoverse-512tvb |	-fopenmp |	-ffast-math
 Nvidia HPC SDK |	23.1	| -O3 -tp=neoverse-v1	| -mp |	-fast
+
+### Commercial HPC applications list for Graviton
+Below is a list of some of the commercial HPC applications that are available on Graviton.
+ISV                   | Application   | Release of support  | Additional Notes
+----------------------|---------------|---------------------|-----------------
+Ansys                 | Fluent        | v221                | [AWS Blog](https://aws.amazon.com/blogs/compute/running-ansys-fluent-on-amazon-ec2-c5n-with-elastic-fabric-adapter-efa)
+Ansys                 | LS-Dyna       | 12.1                | [ISV Article](https://www.ansys.com/content/dam/amp/2023/june/quick-request/)
+Ansys                 | RedHawk-SC    | 2023R1              | [Release Notes](https://www.ansys.com/content/dam/it-solutions/platform-support/arm-64-processor-support-announcement-january-2023.pdf)
+Fritz Haber Institute | FHIaims       | 21.02               | [AWS Blog](https://aws.amazon.com/blogs/hpc/quantum-chemistry-calculation-on-aws/)
+Mathworks             | Matlab        | R2023b              | [Release Notes](https://www.mathworks.com/products/new_products/release-highlights.html), [ISV Article](https://www.mathworks.com/support/requirements/apple-silicon.html)
+Sentieon              | DNAseq , TNseq, DNAscope | 202112.02 | [Release Notes](https://support.sentieon.com/manual/appendix/releasenotes/#release-202112-02), [AWS Blog](https://aws.amazon.com/blogs/hpc/cost-effective-and-accurate-genomics-analysis-with-sentieon-on-aws/)
+Siemens               | StarCCM++     | 2023.2              | [Release Notes](https://blogs.sw.siemens.com/simcenter/simcenter-star-ccm-2302-released/arm-64-processor-support-announcement-june-2023.pdf)
```

Comment:
404 here as well.

Reply:
![404](https://seranking.com/blog/wp-content/uploads/2021/01/404_01-min.jpg)

Reply:
Does this link to the Siemens website require a login of some kind? I'm still getting a 404.

Reply:
Yes, I noticed that right before lunch.  Should all be fixed now.